app/models/product_import/products_reset.rb

```diff
@@ -0,0 +1,35 @@
+module ProductImport
+  class ProductsReset
```

Comment:
I prefer nouns for class names, in this case, ProductsResetter would be better, or ProductsResetStrategy